validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -345,9 +345,14 @@ def _receive_message(self):
                 if self._socket.getsockopt(zmq.TYPE) == zmq.ROUTER:
                     zmq_identity, msg_bytes = \
                         yield from self._socket.recv_multipart()
-                    self._received_from_identity(zmq_identity)
-                    self._dispatcher_queue.put_nowait(
-                        (zmq_identity, msg_bytes))
+                    if msg_bytes == b'':
+			# send ACK for connection probes
```

Comment:
indentation alignment

Reply:
Strange, locally it looks ok...

![image](https://user-images.githubusercontent.com/14056737/48405952-71d8ce00-e71a-11e8-9d53-7b2847b314fe.png)

Reply:
The line is using tabs instead of spaces.